init.sh

```diff
@@ -167,7 +167,7 @@ cd "$DIR_LIB_RT"  && download_url "$EVOSUITE_URL/$EVOSUITE_RT_JAR"
 #
 echo
 echo "Setting up Randoop ... "
-RANDOOP_VERSION="4.2.5"
+RANDOOP_VERSION="4.3.3"
```

Comment:
I would prefer this change to be in a different pull request than the changes to the randoop.sh script.

Reply:
okay - we just have to make sure 4.3.3 comes before the script changes.